vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -88,45 +96,41 @@
      */
     // <editor-fold desc="Configuration">
 
-    private static VimeoClient sharedInstance;
+    private static VimeoClient mSharedInstance;
 
     public static VimeoClient getInstance() {
-        if (sharedInstance == null) {
+        if (mSharedInstance == null) {
             throw new AssertionError("Instance must be configured before use");
         }
 
-        return sharedInstance;
+        return mSharedInstance;
     }
 
     public static void initialize(Configuration configuration) {
-        sharedInstance = new VimeoClient(configuration);
+        mSharedInstance = new VimeoClient(configuration);
     }
 
-    private VimeoClient(Configuration configuration) {
-        if (configuration == null) {
-            throw new AssertionError("Configuration cannot be null");
-        }
-
-        this.configuration = configuration;
-        this.gson = VimeoNetworkUtil.getGson();
-        this.cache = this.configuration.getCache();
-        this.retrofit = createRetrofit();
-        this.vimeoService = retrofit.create(VimeoService.class);
+    private VimeoClient(@NotNull Configuration configuration) {
+        this.mConfiguration = configuration;
```

Comment:
Minor: no longer necessary to use the this keyword here (also could be removed everywhere)